polaris.shopify.com/package.json

```diff
@@ -15,7 +15,8 @@
     "parse-guidelines": "node src/scripts/parseGuidelines.mjs",
     "copy-icons": "node src/scripts/copy-icons.mjs",
     "generate-og-images": "next build && yarn next-sitemap && node src/scripts/generate-og-images.js",
-    "generate-og-images-skip-sitemap": "node src/scripts/generate-og-images.js"
+    "generate-og-images-skip-sitemap": "node src/scripts/generate-og-images.js",
+    "get-props": "tsc ./src/scripts/get-props.ts --target esnext --module node16 --types \"react\" --outDir ./src/scripts/get-props && node ./src/scripts/get-props/scripts/get-props.js && rm -rf ./src/scripts/get-props"
```

Comment:
I am happy to start with this. I think like our other packages `polaris-react` should export metadata and this should be a part of that export. Then we would just import the metadata from that package.